tests/integration/test_auth.py

```diff
@@ -79,21 +79,34 @@ def test_auth_can_create_authenticated_requests_sessions():
     assertions.assertTrue("Bearer" in session.headers["Authorization"])
 
 
-def test_auth_can_fetch_s3_credentials():
+@pytest.mark.parametrize(
+    ("daac", "provider"),
+    (
+        ("NSIDC", "NSIDC_CPRD"),
+        ("GHRCDAAC", "GHRC_DAAC"),
+        ("PODAAC", "POCLOUD"),
+        ("ASF", "ASF"),
+        ("ORNLDAAC", "ORNL_CLOUD"),
+        ("LPDAAC", "LPCLOUD"),
+        ("GES_DISC", "GES_DISC"),
+        ("OBDAAC", "OB_CLOUD"),
+        ("LAADS", "LAADS"),
+        ("ASDC", "LARC_CLOUD"),
+    ),
+)
+def test_auth_can_fetch_s3_credentials(daac, provider):
```

Comment:
We don't need provider here. We can also re-use the pre-existing `earthaccess.daac.DAACS` constant variable for the data structure instead!

Reply:
Are you suggesting to use ```earthaccess.daac.DAACS``` as the variable from which we can access daac and providers, hence it cuts down  the need of having a huge list of parameters ( the ones that I listed above)?

Reply:
Correct, and also, the test never uses the `provider` variable, so it can be removed from the arguments and from the parameters in the decorator :)